src/test/java/plates/DimensionsTest.java

```diff
@@ -29,26 +30,62 @@
         assertEquals(errorMessage, "Number of rows and number of columns cannot be less or equal 0!");
     }
 
-    @Test void compareObjectsWithTheSameDimensions() {
+    @Test
+    void TwentyFourthPlateSizeIsReturnedFor4RowsAnd6Columns() {
```

Comment:
Method names start with lower case letters. You can simplify this to: `plateSizeIs24_for4by6Plate`. We don't usually use `_` in method names, but it's a pretty common convention in tests.